bundles/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/json/obj/Value.java

```diff
@@ -0,0 +1,169 @@
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.unit.SIUnits;
+import org.openhab.core.library.unit.Units;
+import org.openhab.core.thing.type.ChannelTypeUID;
+import org.openhab.core.types.State;
+
+/**
+ * Class holding the Value JSON element
+ *
+ * @author Moritz 'Morty' Strübe - Initial contribution
+ */
+@NonNullByDefault
+public class Value {
+    public double value = 0;
+    public String unit = "-1";
+
+    public String getDesc() {
+        final var el = TYPE_MAP.get(Integer.parseInt(unit));
+        assert el != null : "Type not set or supported";
+        return el.desc;
+    }
+
+    public @Nullable String getType() {
+        final var el = TYPE_MAP.get(Integer.parseInt(unit));
+        assert el != null : "Type not set or supported";
+        return el.unitname;
+    }
+
+    public @Nullable ChannelTypeUID getChannelType() {
+        final var el = TYPE_MAP.get(Integer.parseInt(unit));
+        assert el != null : "Type not set or supported";
+        if (!el.unitname.startsWith("Number")) {
+            return TACmiBindingConstants.CHANNEL_TYPE_SCHEME_STATE_RO_UID;
+        }
+        return TACmiBindingConstants.CHANNEL_TYPE_SCHEME_NUMERIC_RO_UID;
+    }
+
+    public State getState() {
+        final var el = TYPE_MAP.get(Integer.parseInt(unit));
+        assert el != null : "Type not set or supported";
+        if (el.unitname.startsWith("Switch")) {
+            return (value != 0) ? OnOffType.ON : OnOffType.OFF;
+        }
+        if (el.unit != null) {
+            // Must copy to a local variable to make the null-checker happy
+            final var tmp = Objects.requireNonNull(el.unit);
+            return new QuantityType<>(value * el.factor, tmp);
+        }
+        return new DecimalType(Double.valueOf(value * el.factor));
+    }
+
+    static class TypeInfo {
+        public final String desc;
+        public final String unitname;
+        public final @Nullable Unit<@NonNull ?> unit;
+        public final double factor;
+
+        public TypeInfo(String desc, String unitname, @Nullable Unit<@NonNull ?> unit) {
+            this.desc = desc;
+            this.unitname = unitname;
+            this.factor = 1;
+            this.unit = unit;
+        }
+
+        public TypeInfo(String desc, String unitname, double factor, Unit<?> unit) {
+            this.desc = desc;
+            this.unitname = unitname;
+            this.factor = factor;
+            this.unit = unit;
+        }
+    }
+
+    static final Map<Integer, TypeInfo> TYPE_MAP;
+    static {
+     // @formatter:off
+        // -1 is reserved to return null
+        Map<Integer, TypeInfo> lMap = new HashMap<Integer, TypeInfo>();
+        lMap.put(0,  new TypeInfo("Unknown",  "Number:Dimensionless", null));
+        lMap.put(1,  new TypeInfo("°C",       "Number:Temperature",   SIUnits.CELSIUS));
+        lMap.put(2,  new TypeInfo("W/m²",     "Number",               null));
```

Comment:
This can be modelled as `Number:Intensity`
For reference: https://www.openhab.org/docs/concepts/units-of-measurement.html